dbt_platform_helper/domain/versioning.py

```diff
@@ -125,15 +163,8 @@ def get_version_status(self) -> VersionStatus:
 
         return VersionStatus(installed=locally_installed_version, latest=latest_release)
 
-    def get_default_version(self):
-        return (
-            self.config_provider.load_and_validate_platform_config()
-            .get("default_versions", {})
-            .get("platform-helper")
-        )
-
     def get_template_version(self):
-        if self.is_managed():
+        if self.is_auto():
             return self.environment_variable_provider.get(PLATFORM_HELPER_VERSION_OVERRIDE_KEY)
         if self.platform_helper_version_override:
             return self.platform_helper_version_override
```

Comment:
When "auto" is not set, where do we expect to set the value for self.environment_variable_provider.get(PLATFORM_HELPER_VERSION_OVERRIDE_KEY) ?